website/content/community/_index.md

```diff
@@ -166,3 +166,8 @@ visit
 When editing the website, you can preview your changes locally by
 installing [Hugo](https://gohugo.io/) and running `hugo server` from
 the `website` directory.
+
+## Maintainers
+
+More information about the current maintainers of MetalLB, see the [maintainers
+page]({{% relref “maintainers.md” %}}).
```

Comment:
These quotes are causing Hugo errors. We should use standard quotes (or whatever they are called...).

Reply:
done